web-gateway/conf/messages

```diff
@@ -85,3 +85,6 @@ invalid.duration=Please enter a valid duration
 invalid.units=Please enter a valid unit
 
 createUserInstructions=Here you can create a new user who can participate in and create auctions.
+searchInstructions=Here you can search for auctioned items. Search functionality can be enabled by downloading and running a local instance of elasticsearch. \
+	Steps for doing this can be found in README.md.\
+	You can filter your search by specifying the maximum price of items to search for and the currency to search in.
```

Comment:
Hi @mehulsr, thanks for this contribution. 
I have a question: are the newlines preserved when rendering this message in HTML? Or put another way, will this message be rendered as 3 `<p>` blocks? If not, is there 

I think we could rewrite them so that the generic instruction on how to use the feature appear first (since they always apply), and the details on how to get the feature up and running (install ES and/or check README) appear last. Also, I think it's worth mentioning that once the system is started, there's no need to restart it when installing and running ES since the search service will be able to recover.